docs/flink/flink-connector.md

```diff
@@ -141,5 +147,16 @@ SELECT * FROM flink_table;
 +----+------+
 3 rows in set
 ```
+## Lookup Join
```

Comment:
Shouldn't this be next to the Flink SQL content? I don't like that all of Flink SQL is covered in a page called "Getting Started" but I think that section should be broken into two: "SQL" and "DataStream", in which case this will go in the "SQL" section.

Reply:
please check again